lib/temporal/client.rb

```diff
@@ -59,6 +94,14 @@ def start_workflow( # rubocop:disable Metrics/ParameterLists
       implementation.start_workflow(input)
     end
 
+    # Get a workflow handle to an existing workflow by its ID.
+    #
+    # @param id [String] Workflow ID to get a handle to.
+    # @param run_id [String, nil] Run ID that will be used for all calls.
```

Comment:
I'd mention that if omitted, the latest run for a given workflow ID will be referenced.

Reply:
👍